lib/task.js

```diff
@@ -80,6 +91,17 @@ class Task {
 		return Promise.resolve()
 			.then(() => {
 				this.state = state.PENDING;
+				return this.skip();
+			})
+			.then(skipped => {
+				if (skipped) {
+					this.state = state.SKIPPED;
+					if (typeof skipped === 'string') {
+						this._lastLine = stripAnsi(skipped.trim().split('\n').pop().trim());
```

Comment:
Could you remove the `stripAnsi` call at this line and at line 125. I think it would make more sense to call it where the actual logging is done at line 83.